speech/speech_samples.rb

```diff
@@ -12,160 +12,168 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-def speech_sync_recognize project_id: nil, audio_file_path: nil
+def speech_sync_recognize audio_file_path: nil
 # [START speech_sync_recognize]
-  # project_id      = "Your Google Cloud project ID"
   # audio_file_path = "Path to file on which to perform speech recognition"
 
   require "google/cloud/speech"
 
-  speech = Google::Cloud::Speech.new project: project_id
-  audio  = speech.audio audio_file_path, encoding:    :linear16,
-                                         sample_rate: 16000,
-                                         language:    "en-US"
+  speech = Google::Cloud::Speech.new
 
-  results = audio.recognize
+  audio_file = File.binread audio_file_path
+  config     = { encoding: :LINEAR16, sample_rate_hertz: 16000, language_code: "en-US" }
+  audio      = { content: audio_file }
 
-  results.each do |result|
-    puts "Transcription: #{result.transcript}"
+  response = speech.recognize config, audio
+
+  alternatives = response.results.first.alternatives
```

Comment:
What is `results` if it's not already an array of `alternatives`?

*...need to look at recognize result...*

Not great looking... wondering...

Reply:
If `response` only has `results` ... can `response` BE `results`? Maybe? Please? 😺